src/libstd/net/ip.rs

```diff
@@ -231,6 +234,8 @@ impl Ipv4Addr {
     /// - the broadcast address (255.255.255.255/32)
     /// - test addresses used for documentation (192.0.2.0/24, 198.51.100.0/24 and 203.0.113.0/24)
     /// - the unspecified address (0.0.0.0)
+    ///
+    /// [ipv4-sr]: http://goo.gl/RaZ7lg
```

Comment:
I believe it's to please `make tidy`'s line length check. We should find a better way though. If we add `ignore-tidy-linelength` to the file then we can use the full URL but then the whole file won't be checked.